Sources/PointFree/Routes.swift

```diff
@@ -29,7 +29,7 @@ private let routers: [Router<Route>] = [
 
   Route.iso.githubCallback
     <¢> get %> lit("github-auth")
-    %> queryParam("code", .string) <%> queryParam("redirect", opt(.string))
+    %> queryParam("code", opt(.string)) <%> queryParam("redirect", opt(.string))
```

Comment:
I went ahead and made this change, and updated the code to handle it... some interesting things popped out.